test/integration/test_groundlight.py

```diff
@@ -184,6 +184,15 @@ def test_submit_image_query_filename(gl: Groundlight, detector: Detector):
     assert is_valid_display_result(_image_query.result)
 
 
+def test_submit_image_query_with_no_human_review(gl: Groundlight, detector: Detector):
+    # For now, this just tests that the image query is created successfully. There should probably be a better
+    # way to check that the image query is never escalated for human review.
+    _image_query = gl.submit_image_query(detector=detector.id, image="test/assets/dog.jpeg", human_review="NEVER")
+    assert str(_image_query)
+    assert isinstance(_image_query, ImageQuery)
+    assert is_valid_display_result(_image_query.result)
```

Comment:
I think we can just test `assert is_valid_display_result(_image_query.result)` -- the other ones should only be tested in one place.